### PR TITLE
feat(commands): update set sick day to add calendar event

### PR DESCRIPTION
**Pull request by PatrickDohn**
#### Summary

_Enumerate and describe your changes_

- update set sick day command to make a google calendar event.

#### Checklist

_Double check list and complete these actions before merge_

- [x] PR title is in format `TYPE(scope): description`
- [x] Link PR to Issue (if any issues are related)

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ const { RESOURCES, HOLIDAYS } = require('./constants');
 const { COMMANDS, TOPICS } = require('./commands/commands');
 const GoogleSheetUtil = require('./utils/googleSheetsUtil');
 const WebClientUtil = require('./utils/webClientUtil');
+const GoogleCalendarUtil = require('./utils/googleCalendarUtil');
```
Comment (davidholyko) [inline src/app.js:9]: nit import order

```
const GoogleSheetUtil = require('./utils/googleSheetsUtil');
const GoogleCalendarUtil = require('./utils/googleCalendarUtil');
const WebClientUtil = require('./utils/webClientUtil');

```

File: src/utils/googleSheetsUtil.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ const _ = require('lodash');
 const moment = require('moment'); // require
 const { RESOURCES } = require('../constants');
 
-const { GOOGLE_SPREADSHEET_ID, KEY_FILE, SCOPES, HOLIDAYS } = RESOURCES;
```
Comment (davidholyko) [inline src/utils/googleSheetsUtil.js:6]: was `HOLIDAYS` unused?

Comment (PatrickDohn) [reply src/utils/googleSheetsUtil.js:6]: Yes, we only used it in app.js

File: src/utils/googleCalendarUtil.js [hunk 3]
```diff
@@ -1 +1,57 @@
...
+      auth: this.auth,
+    });
+  }
+
+  async createCalendarEvent(slackName) {
+    const today = moment().format('YYYY-MM-DD');
+    const event = {
+      summary: `${slackName}: Sick`,
+      description: 'This event is brought to you by hrbot.',
+      colorId: '11',
```
[the remark this answers is not in the document]
Comment (PatrickDohn) [reply src/utils/googleCalendarUtil.js:27]: I added a comment to reference the google-calendar.md file which now has the event colors table.

**Comment by davidholyko** [comment]
closed as dupe. work done in https://github.com/f1v/hrbot/pull/85

**Pull request closed without merging**